partner_company_type/models/res_partner.py

```diff
@@ -1,5 +1,4 @@
-# -*- coding: utf-8 -*-
-# Copyright 2017 ACSONE SA/NV
+# Copyright 2018 ACSONE SA/NV
```

Comment:
s/2018/2017-2018/

Reply:
@sbidoul IMO to change as `Copyright 2011-TODAY ACSONE SA/NV`